src/util/filteringHttpLogsStream.ts

```diff
@@ -0,0 +1,36 @@
+import { Writable } from "stream";
+
```

Comment:
Not sure why Github didn't pick that one. I renamed filteringStream to better explain what it does + some trivial refactoring.